CONTRIBUTING.md

```diff
@@ -145,9 +145,9 @@ Pandoc is required to render Jupyter Notebooks for building the sphinx documenta
 
 [Pre-commit hooks](https://pre-commit.com/) are strongly encouraged to enforce uniform coding standards across all contributors:
 
-First, install pre-commit within your virtual environment
+First, confirm pre-commit is installed within your virtual environment. It should have been installed via the optional dev dependencies `pip install -e ".[dev]"`.
 ```
-pip install pre-commit
+pip freeze | grep pre-commit
```

Comment:
Could we add a line underneath this to tell the user what to do if pre-commit is not installed? Something like:

If the above command returns empty, run:
`pip install pre-commit`

Reply:
I think I'd prefer users flag an issue with the optional dev dependencies installation rather than encourage a manual install workaround